src/styles/_catalog.scss

```
@@ -1,22 +1,31 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
+  grid-template-rows: 1fr 1fr;
+  gap: 48px;
 
   &__card {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
+}
+
+@media (min-width: 768px) and (max-width: 1023px) {
+  .catalog {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
```

you can use the power of nesting and move media queries in `.catalog`
```suggestion
  }
  
  @media (min-width: 768px) and (max-width: 1023px) {
    grid-template-columns: repeat(3, 200px);
  }
}

```